content/developer/Theme/Icons/SuitePIconsFont/create_Suitepicon_Font.adoc

```diff
@@ -0,0 +1,28 @@
+---
+title: "SuiteP Icons Font"
+weight: 2
+pre: "<b> </b>"
+---
+
+Since 7.9, SuiteCRM has a new font called suitepicon which takes all the icons of SuiteP and turns them into a font. Then means that you can easily change the colors and styling of the icons while keeping them in an optimised format.
+
+To use the icons you need to specify the css classes which are defined in the font:
+
+....
+<span class="suitepicon action-first"></span>
+....
+
+*Locations:*
+
+* themes/SuiteP/css/suitepicon.eot
+* themes/SuiteP/css/suitepicon.html
+* themes/SuiteP/css/suitepicon.json
+* themes/SuiteP/css/suitepicon.scss
+* themes/SuiteP/css/suitepicon.svg
+* themes/SuiteP/css/suitepicon.ttf
+* themes/SuiteP/css/suitepicon.woff
+* themes/SuiteP/css/suitepicon-glyphs.scss
+
+Click https://salesagility.github.io/SuiteP-Icon-Font/index.html[here] to view.
```

Comment:
I once read in a web design manual that we should avoid using `click here` or `here is the link` as link text, unless we're really aiming for a publicity-style call to action. 
Instead, the linked words should be the substance of where the link will take us. This is also good for SEO. 

So, here, you could say something like

These files are available online in the [SuiteP Icon Font GitHub repository](https://salesagility.github.io/SuiteP-Icon-Font/index.html)